lib/src/widgets/keys.dart

```diff
@@ -8,3 +8,4 @@ const kDevicePreviewIcon = Key('device_preview_icon');
 const kRotateIcon = Key('rotate_icon');
 const kHideFrameIcon = Key('hide_frame_icon');
 const kStoryPinIcon = Key('story_pin_icon');
+const kTooltipTitle = Key('tooltip_title');
```

Comment:
I wonder if we could trigger the tooltip without using a key, we may end up with several tooltips on the tree and by setting this key on all of them, they will all have the same key.

Maybe for this, we could find the tooltip by type or text?